one/one/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+using namespace std;
+
+#define endl '\n';
+
+void TwoOne() {
+    const char* msg = "Hello World";
+    cout << msg << endl;
+}
+
```

Comment:
Hello World!
Style: braces should be on a new line:
https://youilabs.atlassian.net/wiki/pages/viewpage.action?pageId=524322#C++CodingStandard-UsingBraces